cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -184,12 +188,18 @@ type Server struct {
 
 var debug = log.New(io.Discard, "", log.LstdFlags)
 
-// our index commands should output something every 100mb they process.
-//
-// 2020-11-24 Keegan. "This should be rather quick so 5m is more than enough
-// time."  famous last words. A client was indexing a monorepo with 42
-// cores... 5m was not enough.
-const noOutputTimeout = 30 * time.Minute
+const (
+	// our index commands should output something every 100mb they process.
+	//
+	// 2020-11-24 Keegan. "This should be rather quick so 5m is more than enough
+	// time."  famous last words. A client was indexing a monorepo with 42
+	// cores... 5m was not enough.
+	noOutputTimeout = 30 * time.Minute
+
+	loggerScope = "server"
+
+	loggerDescription = "periodically reindexes enabled repositories on sourcegraph"
```

Comment:
nit: I'm not sure why these are constants since they are only used in one place?

Reply:
removed